lib/src/serializers/json_api.dart

```diff
@@ -206,6 +206,16 @@ class JsonApiDocument {
             record['type'], record['attributes'], record['relationships']));
   }
 
+  JsonApiDocument? includedDoc(String type) {
+    var id = idFor(type);
+    var it = included
+        .where((record) => record['type'] == type && record['id'] == id)
+        .map<JsonApiDocument>((record) => JsonApiDocument(record['id'],
+        record['type'], record['attributes'], record['relationships']));
+
+    return it.isNotEmpty ? it.first : null;
```

Comment:
@algodave I'm not so familiar with json api requirements, but looks like it makes sense